### PR TITLE
Adding calculator design

### PR DESCRIPTION
**Pull request by duskotmane**
# In this PR I handled the following tasks : 

- [ ] Creating Calculator.js file inside a components folder
- [ ] Create a React component that matches the required design using functional components
- [ ] Import the calculator component  and use it in App.js

### CODE REVIEW

File: src/components/buttons.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <button className="button" type="button">2</button>
+        <button className="button" type="button">3</button>
+        <button className="button operator" type="button">+</button>
+      </div>
+      <div className="row">
+        <button className="button ziro" type="button">0</button>
+        <button className="button" type="button">.</button>
+        <button className="button operator" type="button">=</button>
+        <button className="button equal" type="button">=</button>
+      </div> */}
```
Comment (zieeco) [inline src/components/buttons.js]: - Hey, please, kindly remove these commented pieces of code to make your codebase clean and maintainable.

File: src/components/buttons.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// Button.defaultProps = {
+//   content: 'random',
+// };
+
+// Button.propTypes = {
+//   content: PropTypes.oneOfType([
+//     PropTypes.string,
+//     PropTypes.number,
+//   ]),
+// };
```
Comment (zieeco) [inline src/components/buttons.js]: - Hey, please, kindly remove these commented pieces of code to make your codebase clean and maintainable.

File: src/components/buttons.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```
Comment (zieeco) [inline src/components/buttons.js]: - Hey, please, kindly remove these commented pieces of code to make your codebase clean and maintainable.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @otmaneechchafyky,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Calculator set up correctly. ✔️
- Descriptive PR. ✔️
- Nice design. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @otmaneechchafyky 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by duskotmane** [comment]
@tresorsawasawa Thank you

**Pull request merged**